### DISCUSSION TITLE
How to set a bot as a healer or tank? 怎样将一个机器人设置为治疗或者坦克？

### DISCUSSION THREAD

**Discussion by shieh12** [mod-playerbots/mod-playerbots#639] [Q&A]
Anyone knows, how to set a bot as Healer or Tank?
有人知道，怎么将一个机器人设置为治疗或者坦克么？

Now, I'm using './playerbots bot addclass paladin' to add char, but dont know how to set it as a healer or tank.
what command can'i use to do this?

**Comment by noisiver** [comment]
Whisper them `talents spec holy pve` to make a paladin healer. Whisper them `talents spec list` to see what specs they have available. After changing it it's recommended to also whisper them `reset botAI` but I don't know if that has been fixed.

**Reply by shieh12** [reply]
Thanks for your answer.  I'll have a try later.

**Reply by felipejost** [reply]
Also, if you want them to use specific gear for their role, whisper them `autogear` after you change their talent spec
